gpflow/utilities/multipledispatch.py

```diff
@@ -61,7 +70,7 @@ def get_first_occurrence(self, *types: Types) -> Callable[..., Any]:
         """
         n = len(types)
         for signature in self.ordering:
-            if len(signature) == n and all(map(issubclass, types, signature)):
+            if len(signature) == n and all(map(issubclass, types, signature)):  # type: ignore
```

Comment:
(out of curiosity, what does this complain about?)

Reply:
I forget. `issubclass` had some funny type expectations, and I gave up on making that work with `map` and `Types`.